requirements/requirements_git.txt

```diff
@@ -4,3 +4,4 @@ git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
 git+https://github.com/ansible/python3-saml.git@devel#egg=python3-saml
 django-ansible-base @ git+https://github.com/ansible/django-ansible-base@devel#egg=django-ansible-base[rest_filters,jwt_consumer,resource_registry,rbac]
 awx-plugins-core @ git+https://git@github.com/ansible/awx-plugins.git@devel#egg=awx-plugins-core
+awx_plugins.interfaces @ git+https://github.com/webknjaz/ansible--awx_plugins.interfaces.git@feature/shared-awx-stuff
```

Comment:
This PR is kept in draft until [`37530da`](https://github.com/ansible/awx/pull/15566/commits/37530daf5581c1e9fc182edd93af8f4d5cb7ffd3) is reverted.